src/react-app-env.d.ts

```diff
@@ -1 +1,51 @@
 /// <reference types="react-scripts" />
+
+export interface User {
+  id: number,
```

Comment:
interfaces should have semicolons (`;`) at the end of each property description
```suggestion
  id: number;
```